test/integration/manage_users_test.rb

```diff
@@ -31,7 +31,7 @@ class ManageUsersTest < IntegrationTest
 
     # We need to give jquery a bit of time to let the record fadeout and
     # disappear
-    sleep 0.4
+    sleep 1
```

Comment:
Why does this needs to sleep for 1 second? That more then doubles the integration tests runtime.

Reply:
@jvanbaarsen The 0.4 wasn't enough for running tests inside a container unfortunately. I could try other settings like 0.5 or 0.6.

Reply:
Hm ok. In that case it would be good to move over to the new integration tests from rails 5 at some point, since then this whole dance isn't needed anymore :-) I'll look into that someday

Reply:
Yep ok!